toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -3439,6 +3456,11 @@ void CellArea::createKeyLineMenu(QMenu &menu, int row, int col) {
   if (rType != TDoubleKeyframe::Constant)
     menu.addAction(cmdManager->getAction(MI_UseConstantInterpolation));
```

Comment:
I know this is a little out of scope for this PR, but since you are updating this Context menu, can you move `Constant Interpolation` to the top of the list so it matches the list in the Function Editor?

After more consideration it I also recommend removing `Expression Interpolation` and `File Interpolation` since those options require special input.